draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -272,6 +272,23 @@ exist between `sub` and `cnf` claims.
 Additionally, any public and private claims as defined in Sections 4.2 and 4.3 of
 [@!RFC7519] MAY be used.
 
+If claim values have non-JSON types, such as binary data (e.g., images), they
+MUST be encoded using data URIs as defined in [@!RFC2397].
```

Comment:
Everything binary can be encoded in JSON, the question is just how. Also I'm not sure we should make this a hard requirement. So I would propose to change this to:

```suggestion
Binary data SHOULD be encoded as data URIs as defined in [@!RFC2397]. Exceptions can be made when data formats are used that already define a text encoding suitable for use in JSON or where an established text encoding is commonly used. For example, images would make use of data URIs, whereas hash digests (in hex or base64 encoding) or HTML data do not need to be encoded as such. 
```

Reply:
Perhaps we recommend this for known media types?